pkg/teams/teams.go

```diff
@@ -17,19 +17,24 @@ type Teams struct {
 // Team is a struct describing a single team and its accounts as we
 // expect in the JSON file describing team mappings
 type Team struct {
-	Name     string   `json:"name"`
-	Accounts []string `json:"accounts"`
-	Profiles []string `json:"profiles"`
+	Name     string    `json:"name"`
+	Accounts []Account `json:"accounts"`
 }
 
-// ParseTeamMap takes a path to a team mapping JSON file, reads the file, and returns Go maps of profiles and accounts to team names
-func ParseTeamMap(path string) (profilesToTeams map[string]string, accountsToTeams map[string]string, err error) {
+// Account is a struct describing a single account for a team
+type Account struct {
+	ID          string `json:"id"`
+	Environment string `json:"environment"`
+}
+
+// ParseTeamMap takes a path to a team mapping JSON file, reads the file, and returns a Go map of Accounts to team names
+func ParseTeamMap(path string) (accountsToTeams map[Account]string, err error) {
 	teams, err := readTeamMap(path)
 	if err != nil {
 		return
 	}
 
-	return teams.profilesToTeamNames(), teams.accountsToTeamNames(), err
+	return teams.accountsToTeamNames(), err
```

Comment:
didn't you already check that `err` is `nil`? I think `return teams.accountsToTeamNames(), nil` would be clearer here